packages/tables/resources/views/index.blade.php

```diff
@@ -1467,6 +1467,9 @@ class="fi-ta-header-cell-sort-btn"
                                                     >
                                                         @if (filled($columnHeaderTooltip))
                                                             <span
+                                                                @if ($columnHeaderTooltip instanceof \Illuminate\Contracts\Support\Htmlable)
+                                                                    x-tooltip.html
+                                                                @endif
                                                                 x-tooltip="{
```

Comment:
I might be wrong, but isn't it cleaner in the generated HTML to conditionally add the `.html` modifier to this line instead? Otherwise we have `x-tooltip.html x-tooltip`?